packages/material-ui/src/styles/colorManipulator.js

```diff
@@ -105,19 +105,38 @@ export function decomposeColor(color) {
   const marker = color.indexOf('(');
   const type = color.substring(0, marker);
 
-  if (['rgb', 'rgba', 'hsl', 'hsla'].indexOf(type) === -1) {
+  if (['rgb', 'rgba', 'hsl', 'hsla', 'color'].indexOf(type) === -1) {
     throw new Error(
       [
         `Material-UI: unsupported \`${color}\` color.`,
-        'We support the following formats: #nnn, #nnnnnn, rgb(), rgba(), hsl(), hsla().',
+        'We support the following formats: #nnn, #nnnnnn, rgb(), rgba(), hsl(), hsla(), color().',
       ].join('\n'),
     );
   }
 
-  let values = color.substring(marker + 1, color.length - 1).split(',');
+  let values = color.substring(marker + 1, color.length - 1);
+  let space;
+
+  if (type === 'color') {
+    values = values.split(' ');
+    space = values.shift();
```

Comment:
This is the color space, right? Could you rename this. I was confused for a second thinking this was an actual " " because of the delimiter.

Reply:
Yes of course. You're right